cupy/testing/_pytest_impl.py

```diff
@@ -4,17 +4,20 @@
 
 try:
     import pytest
+    import _pytest
     _error = None
 except ImportError as e:
+    pytest = None
+    _pytest = None
     _error = e
 
 
 def is_available():
-    return _error is None
+    return _error is None and hasattr(pytest, 'fixture')
```

Comment:
If we allow importing non-modules, it looks better to try-import the least requirements for each file than to import `is_available`.

```python
try:
    import pytest
    from pytest import fixture
except ImportError:
    ...
```

Reply:
By this line I intended to make sure that `pytest` is not an empty namespace. (e.g., `hasattr(pytest, 'mark')` can also be used here)

I think `is_available()` and `check_available()` should behave consistently, so if we don't export `is_available()`, `check_available()` should be implemented in every module. And that what I want to avoid.